src/htm/algorithms/Connections.hpp

```diff
@@ -525,17 +525,6 @@ class Connections : public Serializable
    */
   void bumpSegment(const Segment segment, const Permanence delta);
 
-  /**
-   * Destroy the synapses with the lowest permanence values.  This method is
-   * useful for making room for more synapses on a segment which is already
-   * full.
-   *
-   * @param segment - Index of segment in Connections, to be modified.
-   * @param nDestroy - Must be greater than or equal to zero!
-   * @param excludeCells - Presynaptic cells which will NOT have any synapses destroyed.
-   */
-  void destroyMinPermanenceSynapses(const Segment segment, Int nDestroy,
```

Comment:
removed as not entirely bio-inspired, using synaptic decay instead. 

PS: I also hope for some better cross platform determinism, as the simpler code does not depend so much on ordering of synapses.

Reply:
> removed as not entirely bio-inspired, using synaptic decay instead. 

I now know what you're upto, and what was actually missing from this PR! the decay itself. 
I'm gonna implement it for the synapses' updates.